doc/nrfxlib/nrfxlib.doxyfile.in

```diff
@@ -1993,7 +1993,12 @@ PREDEFINED             = "MBEDTLS_CONFIG_FILE=\"@NRFXLIB_BINARY_DIR@/mbedtls_dox
 			 "MBEDTLS_AES_C=" \
 			 "MBEDTLS_CCM_C=" \
 			 "MBEDTLS_GCM_C=" \
-			 "MBEDTLS_CONFIG_FILE="
+			 "MBEDTLS_CONFIG_FILE=" \
+			 "NRF_802154_INTERNAL_RADIO_IRQ_HANDLING=" \
+			 "NRF_802154_USE_RAW_API=" \
+			 "NRF_802154_CSMA_CA_ENABLED=" \
```

Comment:
Does such naming in doxygen configuration allow for parsing for both `=1` and `=0` sides of the source codes?

Reply:
Or it has some other purpose?

Reply:
This is what the docs say:   _The PREDEFINED tag can be used to specify one or more macro names that are defined before the preprocessor is started (similar to the -D option of e.g.  gcc). The argument of the tag is a list of macros of the form: name or name=definition (no spaces). If the definition and the "=" are omitted, "=1" is assumed. To prevent a macro definition from being undefined via #undef or recursively expanded use the := operator instead of the = operator._

Reply:
Thank you for the explanation. Let's stick with the current choice then.

Reply:
Fixed as agreed